src/helper/helper.js

```diff
@@ -182,6 +182,8 @@ helper.DataLayerHelper.prototype.processStates_ =
     var update = this.unprocessed_.shift();
     if (helper.isArray_(update)) {
       helper.processCommand_(update, this.model_);
+    } else if (helper.isArguments_(update)) {
+      helper.processArguments_(update, this.model_);
```

Comment:
Since you're modifying the processStates_ code, you should add some tests to a processStates_test.js file.

Reply:
Im not sure how I could test the new processStates_ code because it doesn't return anything or change the abstract model it just calls the processArguments_ function which is empty, I think the testing for this should come in the next PR!

Reply:
Given the lack of side effects, you'd need to do something like a mock (or in Jasmine, a spy) to test it in the PR.  I think waiting until the function has side-effects is reasonable, though, and wouldn't recommend a mock-based test here.

https://jasmine.github.io/2.5/introduction#section-Spies